opencti-platform/opencti-front/src/utils/utils.ts

```diff
@@ -44,3 +44,40 @@ export const getFileUri = (id: string) => {
 };
 
 export const generateUniqueItemsArray = <T>(submittedArray: IterableIterator<T>) => Array.from(new Set(submittedArray));
+
+export const getCurrentTab = (location: Location, entity_id: string, entity_type_path: string) => {
+  if (location.pathname.includes(`${entity_type_path}/${entity_id}/knowledge`)) return `${entity_type_path}/${entity_id}/knowledge`;
+  if (location.pathname.includes(`${entity_type_path}/${entity_id}/content`)) return `${entity_type_path}/${entity_id}/content`;
+  return location.pathname;
+};
+
+export const getPaddingRight = (location: Location, entity_id: string, entity_type_path: string) => {
```

Comment:
Thanks for refactoring this part: just one small question: since we only use location.pathname, I don't see the use to have the whole Location object in parameter, maybe locationPath is enough ? same question for getCurrentTab.